modules/abstract-utxo/src/parseOutput.ts

```diff
@@ -222,6 +222,29 @@ export async function parseOutput({
   let currentAddressType: string | undefined = undefined;
   const RECIPIENT_THRESHOLD = 1000;
   try {
+    // In the case of PSBTs, we can already determine the internal/external status of the output addresses
+    // based on the derivation information being included in the PSBT. We can short circuit GET v2.wallet.address
+    // and save on network requests. Since we have the derivation information already, we can still verify the address
+    if (currentOutput.external !== undefined) {
```

Comment:
you can then use `isWalletOutput(o)` here

Reply:
not really, if `external === true` we want to go in here as well

Reply:
gotcha